pkg/hooks/hook_env.go

```diff
@@ -79,7 +79,7 @@ func (h *HookEnv) PodFromDisk() (*pods.Pod, error) {
 
 // Initializes a pod based on the hooked pod manifest and the system pod root
 func (h *HookEnv) Pod() (*pods.Pod, error) {
```

Comment:
this function gets imported and called by hooks, it's just convenience around constructing a `Pod` based on environment variables available to the hook. You probably want to pass the result of `HOOKED_POD_READ_ONLY` instead of `false` as the default to the read only policy? it's somewhat hacky since we're not going to actually configure a blacklist and a whitelist but it will only run in the context of the hooked pod so it should be okay

Reply:
Fixed